src/LinearSolvers/iterativeWrapper.jl

```diff
@@ -75,18 +76,21 @@ Optional Inputs:
 """
 function getIterativeSolver(IterMethod::Function;PC=:ssor,maxIter=500,tol=1e-5,
 					Ainv=identity,out=-1,doClear::Bool=true,nthreads::Int=4,sym=0,isTranspose=false)
- 	return IterativeSolver(IterMethod,PC,maxIter,tol,Ainv,out,doClear,nthreads,sym,isTranspose,0,0,.0,.0,.0)
+ 	return IterativeSolver([], IterMethod,PC,maxIter,tol,Ainv,out,doClear,nthreads,sym,isTranspose,0,0,.0,.0,.0)
 end
 
-function solveLinearSystem!(A,B,X,param::IterativeSolver,doTranspose=0)
+function solveLinearSystem!(A, B,X,param::IterativeSolver,doTranspose=0)
+   # A is only needed when initializing.
 	if param.doClear
 		# clear preconditioner
 		clear!(param)
+		param.AA = []
```

Comment:
would be cleaner to include param.AA = [] in the clear! function for IterativeSolver?